app/models/country_band_detail.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class CountryBandDetail < ApplicationRecord
```

Comment:
You can specify a `belongs_to :country_band` association by providing `primary_key: :number` and `foreign_key: :number` I think.

Reply:
Don't confuse these with SQL `PRIMARY KEY` and `FOREIGN KEY`. The options I'm talking about here are (easily misunderstandable) Rails AR vocabulary, which means "the key [i.e. column] where I look up information in the foreign [i.e. pointing to] model, or in myself [i.e. primary]".

Reply:
Okay done

Reply:
Oh wait, removed it for now. But looks like the association is not used anywhere, is it still needed to have this association here?